docs/dg/dev/third-party-module-development/onboard-and-learn.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Onboard & Learn
+description: Onboard & Learn
+last_updated: Jun 7, 2024
+template: howto-guide-template
+
+---
+
+Basic rules of the module development can be found at https://docs.spryker.com/docs/dg/dev/architecture/architectural-convention.html (With **module development** remarks)
+
+Whenever we provide different versions of our module, we need to follow the basic Semantic Versioning rules that can be found at http://semver.org/ with one exceptions:
```

Comment:
mb we can also link this page? https://docs.spryker.com/docs/dg/dev/architecture/module-api/semantic-versioning-major-vs.-minor-vs.-patch-release.html

Reply:
```suggestion
Whenever we provide different versions of our module, we need to follow the basic Semantic Versioning rules that can be found at http://semver.org/ with one exception:
```